source/functions.lua

```diff
@@ -198,33 +198,33 @@ function functions.HandleSockets(dt)
 					msg = {}
 					msg.name = "ConnectionAccepted"
 				else
-					garrLanders[2] = {}			--! super big flaw: this hardcodes garrLanders[2]
-					garrLanders[2].x = incoming.x
-					garrLanders[2].y = incoming.y
-					garrLanders[2].angle = incoming.angle
-					garrLanders[2].name = incoming.name
+					LANDERS[2] = {}			--! super big flaw: this hardcodes LANDERS[2]

```

Comment:
Is this old code? I thought the hard coding of [2] was removed? I'll double check.